docs/README.md

```diff
@@ -85,6 +85,16 @@ We have so far discussed a few different strategies to solve the rendering probl
 
 and it's not just contentState but contentState + some config
 
+## Support
+
+### Python support
+Python versions as defined in `setup.py` classifiers.
+
+#### Which version combinations to include in Travis test matrix?
+All supported Python version should be tested.
+
+At the moment, tests are only ran against the latest version of BeautifulSoup `4.x` and the latest version of LXML.
```

Comment:
I realized that we only test against the latest version of both engines. Do we want to instead define a list of supported versions? At the very least, we should have a lower bound and upper bound and test against each of them.